docs/flow/unidist/config.rst

```diff
@@ -0,0 +1,68 @@
+..
+      Copyright (C) 2021-2022 Modin authors
+
+      SPDX-License-Identifier: Apache-2.0
+
+:orphan:
+
+Unidist Configuration Settings
+""""""""""""""""""""""""""""""
+
+Using these configuration settings, the user can tune unidist's behavior. Below you can find
+configuration settings currently provided by unidist.
+
+Public API
+''''''''''
+
+Potentially, the source of configuration settings can be any, but for now only environment
+variables are implemented. Any environment variable originates from
+:class:`~unidist.config.parameter.EnvironmentVariable`, which contains most of
+the config API implementation.
+
+.. autoclass:: unidist.config.parameter.EnvironmentVariable
+  :members: put, get
+
+Unidist Configuration Settings List
+'''''''''''''''''''''''''''''''''''
+
++-----------------------+----------------------------------+-------------------------------------------------------------------------+
+|| **Config Name**      || **Env. Variable Name**          || **Description**                                                        |
+|| Backend              || UNIDIST_BACKEND                 || Distribution backend to run queries by                                 |
+|| CpuCount             || UNIDIST_CPUS                    || How many CPU cores to use during initialization of the unidist backend |
+|| RayGpuCount          || UNIDIST_RAY_GPUS                || How many GPU devices to use during initialization of the Ray backend   |
+|| IsRayCluster         || UNIDIST_RAY_CLUSTER             || Whether Ray is running on pre-initialized Ray cluster                  |
+|| RayRedisAddress      || UNIDIST_RAY_REDIS_ADDRESS       || Redis address to connect to when running in Ray cluster                |
+|| RayRedisPassword     || UNIDIST_RAY_REDIS_PASSWORD      || What password to use for connecting to Redis                           |
+|| RayObjectStoreMemory || UNIDIST_RAY_OBJECT_STORE_MEMORY || How many bytes of memory to start the Ray object store with            |
+|| DaskMemoryLimit      || UNIDIST_DASK_MEMORY_LIMIT       || How many bytes of memory that Dask worker should use                   |
+|| IsDaskCluster        || UNIDIST_DASK_CLUSTER            || Whether Dask is running on pre-initialized Dask cluster                |
+|| DaskSchedulerAddress || UNIDIST_DASK_SCHEDULER_ADDRESS  || Dask Scheduler address to connect to when running in Dask cluster      |
++-----------------------+----------------------------------+-------------------------------------------------------------------------+
+
+Usage Guide
+'''''''''''
+
+See example of interaction with unidist configs below, as it can be seen config
+value can be set either by setting the environment variable or by using config
+API.
```

Comment:
```suggestion
Сonfig value can be set with the environment variable, using config API or unidist CLI.
```
I suggest to avoid the redundant lexical constructions which doesn't include any additional semantic load.

Also, need to add example of setting config via CLI option.

Reply:
fixed and added